frontend/express/public/core/events/javascripts/countly.overview.views.js

```diff
@@ -1,32 +1,140 @@
             }
-        };
-    },
-    computed: {
-        eventsTableRows: function() {
-            return this.$store.getters["countlyEventsOverview/detailEvents"];
         },
-    },
-    template: '#overview-tables-events'
-});
+        computed: {
+            eventsTableRows: function() {
+                return this.$store.getters["countlyEventsOverview/detailEvents"];
+            },
+        },
+        template: '#overview-tables-events'
+    });
 
-var EventsBreakdownHorizontalTile = countlyVue.views.BaseView.extend({
-    props: {
-        trend: {
-            type: String
+    var OverviewConfigureDrawer = countlyVue.views.create({
+        template: CV.T('/core/events/templates/configureEvents.html'),
+        data: function() {
+            return {
+                saveButtonLabel: CV.i18n('common.done'),
+                selectEvent: '',
+                selectProperty: '',
+            };
         },
-        change: {
-            type: String
+        computed: {
+            title: function() {
+                return CV.i18n("events.overview.events.configure.events");
+            },
+            eventProperties: function() {
+                return this.$store.getters["countlyEventsOverview/eventProperties"];
+            },
+            configureEventsList: function() {
+                return this.$store.getters["countlyEventsOverview/configureEventsList"];
+            },
+            selectedOverviewEvents: {
+                get: function() {
+                    return this.selectEvent;
+                },
+                set: function(selectedItem) {
+                    this.selectEvent = selectedItem;
+                    this.$store.dispatch('countlyEventsOverview/fetchEventProperties', selectedItem);
+                }
+            },
+            selectedOverviewEventProperties: {
+                get: function() {
+                    return this.selectProperty;
+                },
+                set: function(selectedItem) {
+                    this.selectProperty = selectedItem;
+                }
+            }
+        },
+        props: {
+            controls: {
+                type: Object
+            },
+            selectedEvents: {
+                type: Array
+            }
+        },
+        methods: {
+            onClose: function(event) {
+                this.selectEvent = '',
+                this.selectProperty = '',
+                this.$emit("close", event);
+            },
+            onSubmit: function() {
+                var self = this;
+                this.selectedEvents.forEach(function(item, idx) {
+                    self.selectedEvents[idx].order = idx;
+                });
+                this.$store.dispatch('countlyEventsOverview/fetchEditMap', self.selectedEvents);
+            },
+            add: function() {
+                var self = this;
+                var alreadyExists = false;
+                if (this.selectedEvents.length === 12) {
```

Comment:
what is this check for?

Reply:
We can configure max of 12 events for overview. Hence the check :)

Reply:
okay